x-pack/plugins/spaces/server/plugin.ts

```diff
@@ -168,16 +171,14 @@ export class SpacesPlugin
 
     const router = core.http.createRouter<SpacesRequestHandlerContext>();
 
-    initExternalSpacesApi(
-      {
-        router,
-        log: this.log,
-        getStartServices: core.getStartServices,
-        getSpacesService,
-        usageStatsServicePromise,
-      },
-      this.initializerContext.env.packageInfo.buildFlavor
-    );
+    initExternalSpacesApi({
+      router,
+      log: this.log,
+      getStartServices: core.getStartServices,
+      getSpacesService,
+      usageStatsServicePromise,
+      buildFlavor: this.initializerContext.env.packageInfo.buildFlavor,
```

Comment:
`buildFlavor` is needed only in `SpacesClient` eventually, if there is a way to avoid this prop drilling, I would be happy to change.